deltachat-ios/AppDelegate.swift

```diff
@@ -49,13 +51,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // when the app wakes up from "suspended" state
     // (app is in memory in the background but no code is executed, IO stopped)
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        pendingAppInitialization = true;
+        logger.info("➡️ didFinishLaunchingWithOptions")
```

Comment:
that needs to go after `logger.addDestination(console)` - otherwise, logging has not much effect. 

i'd also move the `SDImage` initialisation part a bit down again so that issues there will be logged.